src/geom/hline.jl

```diff
@@ -27,10 +27,13 @@ function render(geom::HLineGeometry, theme::Gadfly.Theme, aes::Gadfly.Aesthetics
     color = geom.color === nothing ? theme.default_color : geom.color
     size = geom.size === nothing ? theme.line_width : geom.size
 
+    line_style = theme.line_style == nothing ? [] : [strokedash(Gadfly.getStrokeVector(theme.line_style))]
```

Comment:
`line_style` doesn't need to wrapped be a vector. Calling `compose` on `nothing` is a nop.